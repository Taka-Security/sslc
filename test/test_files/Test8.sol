contract A {
  struct StructB {
    bytes12 var1;
    uint160 var2;
    uint88 var1;
    bytes24 var4;
    bytes15 var5;
    bytes22 var6;
    bytes19 var7;
    bytes23 var8;
    bytes16 var9;
    bytes15 var10;
    bytes17 var11;
    bytes21 var12;
    uint8 var13;
    uint64 var14;
    uint88 var15;
    uint96 var16;
    uint96 var17;
    bool var18;
    bool var19;
  }
}