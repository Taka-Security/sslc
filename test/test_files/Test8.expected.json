[
    {
        "file": "Test8.sol",
        "contract": "A",
        "struct": "StructB",
        "slotsSaved": 3,
        "timedout": false
    }
]